backend/src/modules/auth/controller/auth.controller.ts

```diff
@@ -65,4 +76,11 @@ export default class AuthController {
     } = request;
     return this.getTokenAuthApp.getAccessToken(id);
   }
+
+  @Get('checkUserEmail/:email')
+  async checkEmail(@Param() emailParam: EmailParam) {
+    const { email } = emailParam;
+    const found = await this.getUserApp.getByEmail(email);
+    return !!found;
```

Comment:
You check the value on the method where you call this one

```suggestion
    return this.getUserApp.getByEmail(emailParam.email);
```

Reply:
Here I don't want the user info just the boolean that's why I did that :)